packages/js/components/src/experimental-select-tree-control/stories/index.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+
+import React, { createElement } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { SelectTree } from '../select-tree';
+import { Item } from '../../experimental-tree-control/types';
+
+const listItems: Item[] = [
+	{ id: '1', name: 'Technology' },
+	{ id: '1.1', name: 'Notebooks', parent: '1' },
+	{ id: '1.2', name: 'Phones', parent: '1' },
+	{ id: '1.2.1', name: 'iPhone', parent: '1.2' },
+	{ id: '1.2.1.1', name: 'iPhone 14 Pro', parent: '1.2.1' },
+	{ id: '1.2.1.2', name: 'iPhone 14 Pro Max', parent: '1.2.1' },
+	{ id: '1.2.2', name: 'Samsung', parent: '1.2' },
+	{ id: '1.2.2.1', name: 'Samsung Galaxy 22 Plus', parent: '1.2.2' },
+	{ id: '1.2.2.2', name: 'Samsung Galaxy 22 Ultra', parent: '1.2.2' },
+	{ id: '1.3', name: 'Wearables', parent: '1' },
+	{ id: '2', name: 'Hardware' },
+	{ id: '2.1', name: 'CPU', parent: '2' },
+	{ id: '2.2', name: 'GPU', parent: '2' },
+	{ id: '2.3', name: 'Memory RAM', parent: '2' },
+	{ id: '3', name: 'Other' },
+];
+
+export const MultipleSelectTree: React.FC = () => {
+	const [ value, setValue ] = React.useState( '' );
+	const [ selected, setSelected ] = React.useState< Item[] >( [] );
+	return (
+		<SelectTree
+			label="Multiple Select Tree"
+			multiple
+			items={ listItems }
+			selected={ selected }
+			getFilteredItems={ ( items, searchValue ) => {
```

Comment:
I'm wondering how it will look like on the CategoryField implementation. Can we leave that to the follow-up PR that implements it on the CategoryField?